### ISSUE TITLE
Menu above anchor 

### ISSUE THREAD

**Issue by android4canada** [skydoves/PowerMenu#44]
Hello 
Great library
I am just wondering if there is a way that I can display the menu Above anchor as opposed to always being below .

Thank you

**Comment by skydoves** [comment]
@android4canada 
Sorry for late reply.
#46  Here is ther solution about your issue.
`showAsAnchor` with x-offest, y0-offest positions. I will be included next publish version.
Thank you for your issue!

**Comment by skydoves** [comment]
@android4canada 
It is published version `2.0.9`.
You can use the following methods.

```java
showAsAnchorLeftTop
showAsAnchorLeftBottom
showAsAnchorRightTop
showAsAnchorRightBottom
showAsAnchorCenter
```

Thank you for your issue!
